### DISCUSSION TITLE
Deceptive Patterns and WCAG

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#61] [General]
The work we did this past fall included deceptive patterns as accessibility issues.  A deceptive pattern is a design decision that deliberately attempts to force a user down a particular path or to trap attention in a way that the user doesn’t want or may be harmful to them. [Deceptive.Design ](https://www.deceptive.design/types) provides a list of this type of pattern. 

Subgroup work included these in our initial list of outcomes for exploration. The rational is that while these patterns harm everyone, these patterns are much bigger barriers with a larger potential for harm for people with cognitive and learning disabilities than for people without cognitive and learning disabilities.  Many also add additional, unnecessary tab stops.

This purpose of this discussion is to decide whether we should continue to explore these outcomes, explore some of them, or leave them out.

**Comment by mraccess77** [comment]
Personally I believe we should include some of them. I don't think we will be able to get all of them accepted - but perhaps there are a few that a wideer group could agree on.

**Comment by nayanecom** [comment]
I believe that the role of the W3C in setting these standards should be carefully considered. While I acknowledge some merits in this proposal, it appears to potentially encourage litigation over practices that are already addressed by existing legislation and regulations concerning unfair practices. It might be more appropriate for regulatory bodies to take the lead in this area.

**Reply by KevinP-NZ** [reply]
WCAG isn't (shouldn't) be about litigation. If these practices disproportionately affect impaired users then they should clearly be called out. The rest of the world (not US) is more interested in a better experience than whether you can be sued.

**Reply by nayanecom** [reply]
Unfortunately, WCAG has been utilized / weaponized as a tool for litigation in the US. That's just a fact.

**Reply by KevinP-NZ** [reply]
That's a problem for the lawyers. WCAG is a tool to promote inclusion and accessibility. There has been too much fear of litigation in WCAG 2 - hence solid guidelines dismissed as too hard to test. We MUST not make that mistake in WCAG 3.

**Reply by nayanecom** [reply]
There are tens of thousands of businesses who disagree and they are not just from the US.

**Reply by KevinP-NZ** [reply]
perhaps they should do something right then. The legislation issue is the tail wagging the dog. If those businesses are worried about being sued then perhaps making their sites accessible is the way ahead (and clamping down on frivolous lawyers)

**Reply by rachaelbradley** [reply]
(chair hat on) This conversation has moved off topic and would need to be continued in a different discussion thread. Please do not continue it here.  Thank you.

**Comment by lauracarlson** [comment]
I agree with Jon Avila. Let's review the proposed outcomes and see what we can get consensus on.

**Comment by colabottles** [comment]
Obviously, as a huge proponent of this and with the research I have done over the years, along with some of the work COGA has done that ties into some the Deceptive Patterns work I have researched, I would strongly approve for future exploration and careful consideration of adding this to the future guidelines.

A lot of the work done already by myself and others and polling the disabled and accessibility community as a whole has shown me that there is a desire for this to be looked into closely.

I am in full support of doing  exploration to see what can and can be used as well as what can be consolidated into one with regard to some patterns.
